lib/activity_flow/activity_flow_line_indicator.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+import 'package:festival_flutterturkiye_org/core/utils/image_assets.dart';
+
+class ActivityFlowLineIndicatorBuilder extends StatelessWidget {
+  const ActivityFlowLineIndicatorBuilder({
+    @required this.index,
+    @required this.isLastIndex,
+    Key key,
+  })  : assert(index != null),
+        assert(isLastIndex != null),
+        super(key: key);
+
+  final int index;
+  final bool isLastIndex;
+
+  @override
+  Widget build(BuildContext context) {
+    if (index.isEven) {
+      return const ActivityFlowLineIndicator(ImageAssets.lineIndicatorTop);
+    } else if (isLastIndex) {
```

Comment:
Why do we check is it last? If it has a high priority, why didn't we put it as a first if.

If it doesn't, why don't we use it with just else block?

Reply:
If the index is the last, we don't add an indicator. Looks good to me.

Reply:
So you need to check is it the last before it is odd or even. 

What if the last index is even? It will return ActivityFlow, isn't it?

Reply:
As in the design, the last index is even (index % 2 == 0) and there is no indicator next to it.
You can check [[use_is_even_rather_than_modulo](https://dart-lang.github.io/linter/lints/use_is_even_rather_than_modulo.html)] lint rule.

Reply:
I just found a bug and fixed it. I sent you a review request. @bgoktugozdemir

Reply:
I just wanted to say that. If you need to check that it is the last index, you should check it first.

Then, check it is even or odd.

In the previous version;

if (index.isEven)
else if (isLastIndex)
else if (index.isOdd)

According to the code above; if the index is 4 and it Is the last index, it will return the value of index.isEven.
But it has to return the value of isLastIndex.

Reply:
That's correct.